AC3.2-MidUnit-6-Exam/DoubleVideoViewController.swift

```diff
@@ -7,25 +7,128 @@
 //
 
 import UIKit
+import AVKit
+import AVFoundation
+import MobileCoreServices
 
-class DoubleVideoViewController: UIViewController, CellTitled {
+//struct Players {
+//    var player: AVPlayer?
+//    var secondPLayer: AVPlayer?
+//    
+//    init(player: AVPlayer) {
+//        self.player = player
+//    }
+//    
+//    init(secondPlayer: AVPlayer) {
+//        self.secondPLayer = secondPlayer
+//    }
+//}
+
+class DoubleVideoViewController: UIViewController, CellTitled, UIImagePickerControllerDelegate, UINavigationControllerDelegate  {
     var titleForCell: String = "Double Video"
 
     @IBOutlet weak var videoContainerTop: UIView!
     @IBOutlet weak var videoContainerBottom: UIView!
+    var player: AVPlayer?
+    var secondPlayer: AVPlayer?
+    var movieUrl: URL?
+  
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         // don't f with this line
         self.edgesForExtendedLayout = UIRectEdge(rawValue: 0)
     }
 
     override func viewDidLayoutSubviews() {
+//        guard let sublayers = self.videoContainerTop.layer.sublayers
```

Comment:
It was possible to set the bounds in here. Commented out code should be removed after final commits. I see it in some other places too.